datacube_alchemist/cli.py

```diff
@@ -204,16 +210,25 @@ def add_missing_to_queue(config_file, queue, dryrun):
 @cli.command()
 @queue_option
 @limit_option
-@click.option("--to-queue", "-t", help="Url of SQS Queue to move to", required=True)
+@click.option("--to-queue", "-t", help="Url of SQS Queue to move to", required=False)
 @dryrun_option
 def redrive_to_queue(queue, to_queue, limit, dryrun):
     """
-    Redrives all the messages from the given sqs queue to the destination
+    Redrives all the messages from the given sqs queue to their source, or the target queue
     """
 
     dead_queue = get_queue(queue)
-    alive_queue = get_queue(to_queue)
-
+    if to_queue:
```

Comment:
Is there any guarantee that the input queue has a corresponding alive queue? e.g. if I passed in a non-deadletter queue.

Reply:
The supplied queue should be the dead queue, so if it doesn't have an alive queue, hopefully it'll raise an exception... could explicitly handle that I guess.

But yeah, you wouldn't want to redrive to a queue if you didn't have a queue to redrive to!